src/utils/RealtimeFeedUtils.js

```diff
@@ -172,8 +87,33 @@ function getDelayInformation(stopID: String, tripID: String, rtf: Object): ?Obje
   };
 }
 
+function getVehicleInformation(
+  routeID: String,
+  tripID: String,
```

Comment:
if these are all nullable, shouldn't they be marked as `?`, or is the `!routeID` check to see if the string is non-empty?

Reply:
oh... i did not know that `?` meant they are nullable.. i shall go do that now thanks